backend/services/prisonerProfileService.js

```diff
@@ -129,6 +130,9 @@ module.exports = ({
 
     const canViewSocLink = Boolean(isSocUser && socDetails)
 
+    const mostRecentAssessment =
+      hasLength(assessments) && assessments.sort((a, b) => b.assessmentDate.localeCompare(a.assessmentDate))[0]
```

Comment:
I think you want a more specific assessment here, one with the `assessmentCode: 'CSR'` . Definitely only ones relating to CSR, as re-categorisation assessments will also be returned in this array.

Reply:
This is what we do in the cell move header:
```const showCsraLink = assessments => {
  return assessments.some(assessment => assessment.assessmentCode === 'CSR' && assessment.assessmentComment)
}```

Reply:
Thank you, didn't realise that.